Tests/app.config

```diff
@@ -20,7 +20,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="WebSocket4Net" publicKeyToken="eb4e154b696bf72a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-0.14.1.0" newVersion="0.14.1.0" />
+        <bindingRedirect oldVersion="0.0.0.0-0.15.2.11" newVersion="0.15.2.11" />
```

Comment:
IIRC, the GDAX brokerage depends on this as well and I thought we had memory leaks when trying to upgrade it. We'll need to perform some stability/memory leak testing with GDAX brokerage before deploying this live.

@jaredbroad  -- do you remember this as well or am I mis-remembering?

Reply:
Well, we have a sslProtocol issue with WebSocket4Net 14.1.0.

Actually, when we're trying to connect to websocket server, the server requires the specfic sslProtocols and I have found no way to configure it in WebSocket4Net 14.1.0.

If you can help me with this, I'll happy to use 14.1.0.

Reply:
I've made a note to research this further. Thank you for the additional information.

Reply:
I've looked at the GDAX implementation and it seems like they used WebSocketSharp instead of WebSocket4Net.